### PR TITLE
[Mod] showing whitelisted and blacklisted users

### PR DESCRIPTION
**Pull request by Jan200101**
Added a little clode block to show which users are blacklisted or whitelisted.
I haven't decided if showing the name is enough or to show the nickname and maybe the discriminator.

### CODE REVIEW

File: cogs/mod.py [hunk 1]
```diff
@@ -1003,7 +1003,13 @@ def check(m):
     async def blacklist(self, ctx):
         """Bans user from using the bot"""
         if ctx.invoked_subcommand is None:
+            name = list(set([x.name for x in self.bot.get_all_members() if x.id in self.blacklist_list]))
             await send_cmd_help(ctx)
+            if name:
+                userlist = ', '.join(name)
+            else:
+                userlist = "No one"
+            await self.bot.say(box('Blacklisted users:\n' + userlist))
```
Comment (Jan200101) [inline cogs/mod.py]: Unsure if what to display ontop.
My suggestions are

- Blacklisted users :
- Blacklist :

File: cogs/mod.py [hunk 2]
```diff
@@ -1037,7 +1043,13 @@ def check(m):
     async def whitelist(self, ctx):
         """Users who will be able to use the bot"""
         if ctx.invoked_subcommand is None:
+            name = list(set([x.name for x in self.bot.get_all_members() if x.id in self.whitelist_list]))
             await send_cmd_help(ctx)
+            if name:
+                userlist = ', '.join(name)
+            else:
+                userlist = "No one"
+            await self.bot.say(box('Whitelisted users:\n' + userlist))
```
Comment (Jan200101) [inline cogs/mod.py]: Unsure if what to display ontop.
My suggestions are

Whitelisted users :
Whitelist :

File: cogs/mod.py [hunk 3]
```diff
@@ -1003,7 +1003,13 @@ def check(m):
     async def blacklist(self, ctx):
         """Bans user from using the bot"""
         if ctx.invoked_subcommand is None:
+            name = list(set([x.name for x in self.bot.get_all_members() if x.id in self.blacklist_list]))
             await send_cmd_help(ctx)
+            if name:
+                userlist = ', '.join(name)
+            else:
+                userlist = "No one"
```
Comment (Jan200101) [inline cogs/mod.py]: Better text for when no one is blacklisted

File: cogs/mod.py [hunk 4]
```diff
@@ -1037,7 +1043,13 @@ def check(m):
     async def whitelist(self, ctx):
         """Users who will be able to use the bot"""
         if ctx.invoked_subcommand is None:
+            name = list(set([x.name for x in self.bot.get_all_members() if x.id in self.whitelist_list]))
             await send_cmd_help(ctx)
+            if name:
+                userlist = ', '.join(name)
+            else:
+                userlist = "No one"
```
Comment (Jan200101) [inline cogs/mod.py]: Better text for when no one is whitelisted

**Review by Jan200101 — COMMENTED** [review]
I need some feedback for the string I marked and what text to fill in these

**Comment by Twentysix26** [comment]
`Client.get_user_info` is supposed to be used as a last resort. The rate limit is one request per second.

**Comment by Jan200101** [comment]
Adding a exception and using it if the bot isnt on any server with that user

**Comment by Jan200101** [comment]
Added if statement instead of except

**Comment by Twentysix26** [comment]
Adding it in #701

**Pull request closed without merging**